src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -71,7 +71,7 @@
 register_cli_argument('group deployment operation show', 'operation_ids', nargs='+', help='A list of operation ids to show')
 register_cli_argument('group export', 'include_comments', action='store_true')
 register_cli_argument('group export', 'include_parameter_default_value', action='store_true')
-register_cli_argument('group create', 'resource_group_name', completer=None)
+register_cli_argument('group create', 'rg_name', options_list=('--name', '-n'), help='name of the new resource group', completer=None)
```

Comment:
FYI, for other renaming on vm/vmss/vmss, my recent push has got rid of it. I decided to handle them in one place, that for `create` command, the `name` part will not load the default configured value. This is not a new thing, we did the same on splitting `id-part`